recipe/meta.yaml

```diff
@@ -3,16 +3,16 @@
 
 package:
   name: {{ name|lower }}
-  version: v{{ version }}
+  version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/b/birdhouse-birdy/birdhouse-birdy-{{ version }}.tar.gz
   sha256: 729fce08c0eece52c6b0cdeef6a91fba39fcedfa96bad887fb542e276746f6cf
 
 build:
   noarch: python
-  number: 0
-  script: {{ PYTHON }} -m pip install . -vv
+  number: 1
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
```

Comment:
Why --no-deps? just curious.

Reply:
It is also used in the pywps recipe:
https://github.com/conda-forge/pywps-feedstock/blob/master/recipe/meta.yaml